frontend/src/MessageBoard/components/MessageBoard.js

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import Message from './Message';
+
+const Messageboard = styled.div`
+	max-width: 800px;
+	width: 100%;
+	margin: 0 auto;
+	background-color: white;
+	color: black;
+
+	& p {
+		font-size: 16px;
+	}
+`;
+const TopSection = styled.section`
+	margin-left: 8%;
+	margin-right: 8%;
+	top: 20px;
+	right: 18%;
+	position: absolute;
+`;
+
+const StyledLink = styled(Link)`
+	color: black;
+	text-decoration: none;
+	margin: 5px;
+	font-weight: bold;
+`;
+
+const TeamName = styled.div`
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	margin-top: 40px;
+`;
+
+const Teamlogo = styled.div`
+	display: flex;
+	align-items: center;
+
+	& button {
+		height: 50%;
+		margin-left: 5px;
+	}
+`;
+
+const Logo = styled.img`
+	display: inline;
+	margin-right: 5px;
+	border-radius: 45px;
+`;
+
+const MessagesContainer = styled.div`
+	margin: 30px 20px;
+	border: 1px solid black;
+`;
+
+class MessageBoard extends React.Component {
+	constructor() {
+		super();
+		this.state = {
```

Comment:
Not gonna knock y'all for this at the moment, but be sure to move test data from local to redux state if that's how you're going to manage it in the future.